mod-source-record-storage-server/src/main/java/org/folio/services/util/EventHandlingUtil.java

```diff
@@ -71,11 +71,12 @@ private EventHandlingUtil() { }
   public static KafkaProducerRecord<String, String> createProducerRecord(String eventPayload, String eventType,
```

Comment:
@RuslanLavrov  I think it would be better to change this method. Remove the last argument and remove zipping logic inside this one.

Reply:
@VRohach, no, to avoid breaking changes since this method also is used by quick mark editor functionality that uses compressed payload https://github.com/folio-org/mod-source-record-storage/blob/037c6211ba27e4448513ca841d290ff87dffa5cd/mod-source-record-storage-server/src/main/java/org/folio/services/QuickMarcKafkaHandler.java#L121